tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -823,10 +823,33 @@ def complex_agg(x, agg):
     res = [complex_agg(fft[config["coeff"]], config["attr"]) if config["coeff"] < len(fft)
            else np.NaN for config in param]
     index = ['coeff_{}__attr_"{}"'.format(config["coeff"], config["attr"]) for config in param]
+
```

Comment:
This can be removed (possibly a development artefact)

Reply:
Yes, remove please.